src/Core/ServiceWrapper_dotNET4/manifest.xml

```diff
@@ -0,0 +1,10 @@
+﻿<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
+  <assemblyIdentity version="1.0.0.0" name="winsw" type="win32"/>
```

Comment:
Version needs to be patched in the build flow if you want to include it to the release

Reply:
Removed the file should not been created.